src/screens/login/styles/LoginScreenStyles.ts

```diff
@@ -0,0 +1,22 @@
+import {StyleSheet} from 'react-native';
+// import { theme } from '../../../../App';
+import {BASE_DIMENSION} from '../../../core/themes/dimensions'
+
+export const styles = (backgroundProp) =>  StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: 'black',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  logo: {
+    height: 100,
+    width: 140,
+  },
+  marginTop: {
+    marginTop: BASE_DIMENSION * 2 + 4,
```

Comment:
we don't need to make it pixel perfect. it's ok if we leave BASE_DIMENSION * 2, if it looks good